examples/jenkins/jenkins-ephemeral-template.json

```diff
@@ -159,6 +192,35 @@
         "type": "ClusterIP",
         "sessionAffinity": "None"
       }
+    },
+    {
+       "kind": "Service",
+       "apiVersion": "v1",
+       "metadata": {
+         "name": "${JENKINS_SERVICE_NAME}",
+         "annotations": {
+           "service.alpha.openshift.io/dependencies": "[{\"name\": \"jenkins-jnlp\", \"namespace\": \"\", \"kind\": \"Service\"}]",
+           "service.openshift.io/infrastructure": "true"
+         },
+         "creationTimestamp": null
+       },
+       "spec": {
+         "ports": [
+           {
+             "name": "web",
+             "protocol": "TCP",
+             "port": 80,
```

Comment:
because people expect their jenkins server to be running on port 80 (for cases where they access it via the service and not the route).  (And the container can't listen on 8080 because it would have to be privileged)